lbz/type_defs.py

```diff
@@ -0,0 +1,40 @@
+"""
+This code is only for static type checking.
+https://docs.aws.amazon.com/lambda/latest/dg/python-context.html
+"""
```

Comment:
Newer version of pylint will point that out in the future anyway:
```suggestion
"""This code is only for static type checking.

https://docs.aws.amazon.com/lambda/latest/dg/python-context.html
"""
```